### PR TITLE
Generate documentation with sphinx

### PR DESCRIPTION
**Pull request by medgyes**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    'sphinx.ext.duration',
+    'sphinx.ext.doctest',
+    'sphinx.ext.autodoc',
+    'sphinx.ext.autosummary',
+    'sphinx_autodoc_typehints',
+]
+
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../..'))
```
Comment (IstvanM) [inline docs/conf.py:29]: What is the story behind this?

**Comment by IstvanM** [comment]
LGTM, what should we keep in the README and what in the DOCS? 
(My opinion README should most likely contain a bunch of links and info about the project, maybe a quick start guide)

**Review by IstvanM — DISMISSED** [review]

File: docs/conf.py (hunk 1 above)
Comment (medgyes) [reply docs/conf.py]: nice catch, it's just a leftover from an uncommited version where the conf.py and all rst files were in a bit deeper folder structure along with a separate Makefile, now it's not needed anymore

**Comment by medgyes** [comment]
> LGTM, what should we keep in the README and what in the DOCS? (My opinion README should most likely contain a bunch of links and info about the project, maybe a quick start guide)

Next let's get the docs generated and published then we can replace the current README and DOCS with a bunch of links.

**Review by IstvanM — APPROVED** [review]

**Pull request merged**